packages/cosmic-swingset/src/anylogger-agd.js

```diff
@@ -4,15 +4,54 @@ import {
   getEnvironmentOption,
 } from '@endo/env-options';
 import anylogger from 'anylogger';
+import util from 'util';
 
-// Turn on debugging output with DEBUG=agoric
+//#region Golang zerolog-like output affordances
+let logFormat = 'plain';
+export const setLogFormat = format => {
+  logFormat = format;
+};
 
-const DEBUG_LIST = getEnvironmentOptionsList('DEBUG');
+let shortDateTimeFormatter;
+const plainTime = d => {
+  if (!shortDateTimeFormatter) {
+    shortDateTimeFormatter = new Intl.DateTimeFormat(undefined, {
```

Comment:
This requests formatting dependent upon host locale, which we almost certainly don't want. Even using a specific locale isn't guaranteed to be stable over time until https://github.com/tc39/proposal-stable-formatting makes it in.

Reply:
when in Rome...